editor/export_template_manager.cpp

```diff
@@ -195,7 +195,7 @@ void ExportTemplateManager::_install_from_file(const String &p_file) {
 			data.resize(info.uncompressed_size);
 
 			//read
-			ret = unzOpenCurrentFile(pkg);
+			unzOpenCurrentFile(pkg);
```

Comment:
Note that this will trigger another warning about return argument unused. It looks like code analysis tool want to force us to do result-checking even if we don't care about it :)